pennylane_cirq/cirq_device.py

```diff
@@ -226,3 +244,10 @@ def apply(self, operations, **kwargs):
         # Diagonalize the given observables
         for operation in rotations:
             self._apply_operation(operation)
+
+    def define_wire_map(self, wires):
```

Comment:
Trying to get `ThreeDQubit`s to work uncovered a lot of uncovered edge cases at the interaction point between PennyLane's wires and Cirq's qubits